doc/structural.xxml

```diff
@@ -150,6 +150,10 @@
 <!-- doxrox-include igraph_to_prufer -->
 </section>
 
+<section id="forest"><title>Forest</title>
```

Comment:
I'd rather put the `igraph_is_forest()` function in the Trees section for the time being. I don't know how many forest-related functions we are going to have, but forests and trees are closely related anyway so we can keep them in the same section. Just rename the section to "Trees and forests"